config/v2params.json

```diff
@@ -0,0 +1,53 @@
+{
+    "insert":
+    {
+        "assertion when inserting record into table with valid values": [{"Const": "tt1001526", "Your Rating": 6, "Date Rated": "2017-03-18 00:00:00", "Title": "Megamind", "URL": "https://www.imdb.com/title/tt1001526/", "Title Type": "movie", "IMDb Rating": 7.3, "Runtime (mins)": 95.0, "Year": 2010, "Genres": "Animation, Action, Comedy, Family, Sci-Fi", "Num Votes": 208264, "Release Date": "10/28/2010", "Directors": "Tom McGrath"}, null],
+        "assertion when inserting record into table with valid values and null rating": [{"Const": "tt1790454", "Your Rating": null, "Date Rated": "2017-03-18 00:00:00", "Title": "Tom and jerry", "URL": "https://www.imdb.com/title/tt1001526/", "Title Type": "movie", "IMDb Rating": 7.3, "Runtime (mins)": 95.0, "Year": 2010, "Genres": "Animation, Action, Comedy, Family, Sci-Fi", "Num Votes": 208264, "Release Date": "10/28/2010", "Directors": "Tom McGrath"}, null],
+        "assertion when inserting record into table with null values for all columns": [{"Const": null, "Your_Rating": null, "Date Rated": null, "Title": null, "URL": null, "Title Type": null, "IMDb Rating": null, "Runtime (mins)": null, "Year": null, "Genres": null, "Num Votes": null, "Release Date": null, "Directors": null}, -1],
+        "assertion when inserting record into table with title only": ["Megamind", -1],
```

Comment:
lines 7-10 are duplicates. what is the overall scenario? have a case for that only, i.e. primary key missing